x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/rule_monitoring/detection_engine_health/get_rule_health/get_rule_health_route.ts

```diff
@@ -77,4 +78,9 @@ export interface GetRuleHealthResponse {
    * Result of the health stats calculation.
    */
   health: RuleHealthSnapshot;
+
+  /**
+   * Summary of the gaps in the rule execution.
+   */
+  gap_summary: GapSummary;
```

Comment:
The new added code (both this addition to the API contract and the implementation added to the route handler) do not follow the existing API design where we have `RuleHealthSnapshot` type that is supposed to contain all the stats and state returned by the API endpoint.

Can we refactor the code according to the API design @denar50?

Let me know if you need any more specific guidance.